pvlib/bifacial/infinite_sheds.py

```diff
@@ -0,0 +1,831 @@
+        apparent_zenith, azimuth, surface_azimuth)
+    # length of shadow behind a row as a fraction of pitch
+    x = gcr * (sind(surface_tilt) * tan_phi + cosd(surface_tilt))
+    f_x = 1 - 1. / x
+    # set f_x to be 1 when sun is behind the array
+    ao = aoi(surface_tilt, surface_azimuth, apparent_zenith, azimuth)
+    f_x = np.where(ao < 90, f_x, 1.)
+    # when x < 1, the shadow is not long enough to fall on the row surface
+    f_x = np.where(x > 1., f_x, 0.)
+    return f_x
+
+
+def get_irradiance_poa(surface_tilt, surface_azimuth, apparent_zenith,
+                       azimuth, gcr, height, pitch, ghi, dhi, dni,
+                       albedo, iam=1.0, npoints=100):
+    r"""
+    Calculate plane-of-array (POA) irradiance on one side of a row of modules.
+
+    The infinite sheds model [1] assumes the PV system comprises parallel,
+    evenly spaced rows on a level, horizontal surface. Rows can be on fixed
+    racking or single axis trackers. The model calculates irradiance at a
+    location far from the ends of any rows, in effect, assuming that the
+    rows (sheds) are infinitely long.
+
+    POA irradiance components include direct, diffuse and global (total).
+    Irradiance values are reduced to account for reflection of direct light,
+    but are not adjusted for solar spectrum or reduced by a module's
+    bifaciality factor.
+
+    Parameters
+    ----------
+    surface_tilt : numeric
+        Tilt of the surface from horizontal. Must be between 0 and 180. For
+        example, for a fixed tilt module mounted at 30 degrees from
+        horizontal, use ``surface_tilt=30`` to get front-side irradiance and
+        ``surface_tilt=150`` to get rear-side irradiance. [degree]
+
+    surface_azimuth : numeric
+        Surface azimuth in decimal degrees east of north
+        (e.g. North = 0, South = 180, East = 90, West = 270). [degree]
+
+    apparent_zenith : numeric
+        Refraction-corrected solar zenith. [degree]
+
+    azimuth : numeric
+        Solar azimuth. [degree]
+
+    gcr : float
+        Ground coverage ratio, ratio of row slant length to row spacing.
+        [unitless]
+
+    height : float
+        Height of the center point of the row above the ground; must be in the
+        same units as ``pitch``.
+
+    pitch : float
+        Distance between two rows; must be in the same units as ``height``.
+
+    ghi : numeric
+        Global horizontal irradiance. [W/m2]
+
+    dhi : numeric
+        Diffuse horizontal irradiance. [W/m2]
+
+    dni : numeric
+        Direct normal irradiance. [W/m2]
+
+    albedo : numeric
+        Surface albedo. [unitless]
+
+    iam : numeric, default 1.0
+        Incidence angle modifier, the fraction of direct irradiance incident
+        on the surface that is not reflected away. [unitless]
+
+    npoints : int, default 100
+        Number of points used to discretize distance along the ground.
+
+    Returns
+    -------
+    output : OrderedDict or DataFrame
```

Comment:
A few years later, perhaps we just go with `dict` instead of `OrderedDict`?

Reply:
@wholmgren do you mean change only the docstring, or replace the use of `OrderedDict` with `{}`?

Reply:
Replace the use